### PR TITLE
Gives Account Registration Device unique sprite and fixes its issues  

### PR DESCRIPTION
**Pull request by OneAsianTortoise**
## About The Pull Request
**Fixes being able to use account registration device without power. 
Fixes being unable to un-anchor or screw open account registration device. 
Gives account registration device a unique sprite instead of the recharger sprite.
Edit: Account registration device says now when used.**
Sprite

https://user-images.githubusercontent.com/64306407/152635354-d6272e11-4dec-480c-880d-4eb3ae03d91a.mp4

I wanted to practice coding and drawing sprite, so I picked one of the most unused and broken machine in the game to do it. 
The code might be a mess, and the sprite might not be up to standard, but I hope this works.

## Why It's Good For The Game
You can now use tools normally on account registration device instead of having to deconstruct it with brute force.
Also, it has a unique sprite now.

## Changelog

:cl:
fix: fixes not being able to use tools on account registration device and it functioning without power
imageadd: gives account registration device a unique sprite instead of weapon recharger sprite
/:cl:

### CODE REVIEW

**Comment by JohnFulpWillard** [comment]
just an idea, it could balloon alert when it makes a new bank account and automatically enter the memories of the person who registers it?

**Comment by OneAsianTortoise** [comment]
> just an idea, it could balloon alert when it makes a new bank account and automatically enter the memories of the person who registers it?

Yeah sure. I can try.

**Comment by OneAsianTortoise** [comment]
I added balloon alert, but I didn't add memory as I don't want HoP's memories full of new account ID
(Also I don't know how to properly add new account ID memory without it replacing old one.

image

**Comment by OneAsianTortoise** [comment]
I added balloon alert, but I didn't add memory as I don't want HoP's memories full of new account ID
(Also I don't know how to properly add new account ID memory without it replacing old one.

image

File: code/game/machinery/accounting.dm [hunk 1]
```diff
@@ -36,8 +36,19 @@
 			bank_account.account_job = /datum/job/unassigned
 		playsound(loc, 'sound/machines/synth_yes.ogg', 30 , TRUE)
 		to_chat(user, span_notice("New account registered under account identification number [bank_account.account_id]."))
+		balloon_alert(user, "new account [bank_account.account_id] registered")
```
Comment (JohnFulpWillard) [inline code/game/machinery/accounting.dm:39]: don't balloon alert the account ID please, just that a new account was registered. Even better, just make the machine say it, and say will handle the balloon alert.

**Review by Imaginos16 — APPROVED** [review]
Cute sprite, nice job!

**Pull request merged**